src/transform/deadCode.ml

```diff
@@ -0,0 +1,217 @@
+open Batteries
+open GoblintCil
+open GobConfig
+open MyCFG
+
+let dpf fmt = Pretty.gprintf (fun doc -> print_endline @@ Pretty.sprint ~width:Int.max_num doc) fmt
+
+let empty_block () = { battrs = [] ; bstmts = [] }
```

Comment:
This doesn't need to be a function:
```suggestion
let empty_block = { battrs = [] ; bstmts = [] }
```

Reply:
`battrs` and `bstmts` are mutable, hence the function. Currently, these empty blocks are never modified further, so it wouldn't matter, but sharing them seems like a potential future headache.

Reply:
Ah, that's right! A function is indeed safer then. Might be worth adding a comment about it such that nobody in the future is tempted to do the same optimization.